frontend/src/components/OdhAppCard.tsx

```diff
@@ -92,7 +92,7 @@ const OdhAppCard: React.FC<OdhAppCardProps> = ({ odhApp }) => {
   ];
 
   const badgeClasses = classNames('odh-card__partner-badge', {
-    'm-warning': odhApp.spec.category === 'Third party support',
+    'm-warning': odhApp.spec.support === 'third party support',
```

Comment:
I'm concerned about these support vs category changes -- also any reason they are not an enum?